payments/models.py

```diff
@@ -0,0 +1,83 @@
+
+import uuid
+from django.core.urlresolvers import reverse
+from django.db import models
+import getpaid
+
+
+CURRENCIES = (
+    ('DKK', 'DKK'),
+    ('PLN', 'PLN'),
+    ('GBP', 'GBP'),
+)
+
+
+class OrderManager(models.Manager):
+
+    def create_for_fee(self, user, fee):
+        name = "Order for {}".format(fee.name)
+        instance = self.model(name=name,
+                              total=fee.amount,
+                              user=user,
+                              currency=fee.currency)
+        instance.save()
+        return instance
+
+
+class Order(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Comment:
Overwriting the auto incremented numeric ID is never a good idea, but having a unique=True additional field for UUIDs is.